mods/common/languages/en.ftl

```diff
@@ -85,7 +84,472 @@ requires-forum-account = Server requires players to have an OpenRA forum account
+news-parsing-failed = Failed to parse news: { $message }
+
+## MapChooserLogic
+all-maps = All Maps
+no-matches = No matches
+player-players =
+    { $players ->
+        [one] { $players } Player
+       *[other] { $players } Players
+    }
+map-size-huge = (Huge)
+map-size-large = (Large)
+map-size-medium = (Medium)
+map-size-small = (Small)
+
+map-deletion-failed = Failed to delete map '{ $map }'. See the debug.log file for details.
+delete-map-title = Delete map
+delete-map-prompt = Delete the map '{ $title }'?
+delete-map-accept = Delete
+
+delete-all-maps-title = Delete maps
+delete-all-maps-prompt = Delete all maps on this page?
+delete-all-maps-accept = Delete
+
+## MissionBrowserLogic
+no-video-title = Video not installed
+no-video-text = The game videos can be installed from the
+    "Manage Content" menu in the mod chooser.
+no-video-cancel = Back
+
+## MusicPlayerLogic
+sound-muted = Audio has been muted in settings.
+no-song-playing = No song playing
+
+## MuteHotkeyLogic
+audio-muted = Audio muted.
+audio-unmuted = Audio unmuted.
+
+## PlayerProfileLogic
+loading-player-profile = Loading player profile...
+loading-player-profile-failed = Failed to load player profile.
+
+## ReplayBrowserLogic
+duration = Duration
+singleplayer = Singleplayer
+multiplayer = Multiplayer
+
+today = Today
+last-week = Last 7 days
+last-fortnight = Last 14 days
+last-month = Last 30 days
+
+replay-duration-very-short = Under 5 min
+replay-duration-short = Short (10 min)
+replay-duration-medium = Medium (30 min)
+replay-duration-long = Long (60+ min)
+
+rename-replay-title = Rename Replay
+rename-replay-prompt = Enter a new file name:
+rename-replay-accept = Rename
+
+delete-replay-title = Delete selected replay?
+delete-replay-prompt = Delete replay { $replay }?
+delete-replay-accept = Delete
+
+delete-all-replays-title = Delete all selected replays?
+delete-all-replays-prompt =
+    { $count ->
+        [one] Delete { $count } replay.
+       *[other] Delete { $count } replays.
+    }
+delete-all-replays-accept = Delete All
+
+replay-deletion-failed = Failed to delete replay file '{ $file }'. See the debug.log file for details.
+
+## ReplayUtils
+incompatible-replay-title = Incompatible Replay
+incompatible-replay-prompt = Replay metadata could not be read.
+-incompatible-replay-recorded = It was recorded with an
+incompatible-replay-unknown-version = { -incompatible-replay-recorded } unknown version: { $version }.
+incompatible-replay-unknown-mod = { -incompatible-replay-recorded } unknown mod: { $mod }.
+incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } unavailable mod: { $mod }.
+incompatible-replay-incompatible-version = { -incompatible-replay-recorded } incompatible version: { $version }.
+incompatible-replay-unavailable-map = { -incompatible-replay-recorded } unavailable map: { $map }.
+
+## ServerCreationLogic
+internet-server-nat-A = Internet Server (UPnP/NAT-PMP
```

Comment:
There's an opening bracket here, but no closing bracket?

Reply:
It is cut off here because multiple text instances are used to write labels in different colors. I added the -C part now which contains the closing parenthesis to not confuse translators.